lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
```

Comment:
Especially if we're only supporting Global Sidebar layouts (but potentially also even if we're supporting Legacy layouts) I'm wondering if it would be cleaner to have the toast container be a sibling of `NxPageMain` rather than a child.  Put another way, to have the toast container be a child of `.nx-page`.  I'd imagine that if it were done that way the toasts could be easily positioned using the grid defined by `.nx-page` rather than this complex combination of absolute positioning, fixed positioning, and flexbox.  I strongly suspect that we could even get rid of `.nx-toast__wrapper` in that case and only have `.nx-toast__container`

Reply:
> Especially if we're only supporting Global Sidebar layouts (but potentially also even if we're supporting Legacy layouts) I'm wondering if it would be cleaner to have the toast container be a sibling of `NxPageMain` rather than a child. Put another way, to have the toast container be a child of `.nx-page`. I'd imagine that if it were done that way the toasts could be easily positioned using the grid defined by `.nx-page` rather than this complex combination of absolute positioning, fixed positioning, and flexbox. I strongly suspect that we could even get rid of `.nx-toast__wrapper` in that case and only have `.nx-toast__container`

NxToastContainer is now a direct child of `nx-page` instead of `NxPageMain`, implemented here: [9c05518](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/9c05518fc4f0e5bcdfa1577d3c5be1f1856fc14c)
. By doing so, `nx-toast__wrapper` has been removed from the code. This led to some minor adjustments needing to be made in the animation to prevent weird horizontal scrolling behaviour on entry (addition of overflow:hidden), and removed the margin values from the animation as they were creating a small bounce.